jython/InitAllTurnoutsSensors.py

```diff
@@ -45,8 +45,8 @@ class InitializeTurnoutsSensors(jmri.jmrit.automat.AbstractAutomaton):
     log = LoggerFactory.getLogger("jmri.jmrit.jython.exec.script.InitializeTurnoutsSensors");
     #    All messages written to self.log.info( "text" ) are
     #        visible in the JMRI System console window at runtime and 
-    #        are appended to file: preference:systemfiles\messages.log
-    #        are stored in file: preference:systemfiles\session.log
+    #        are appended to file: preference:log\messages.log
```

Comment:
"preference:" should be "settings:".  "preference:" is the user files location.

Reply:
Good spot, thanks @dsand47 , have updated.